packages/compass-export-to-language/src/stores/store.js

```diff
@@ -10,6 +10,31 @@ import {
 } from '@mongodb-js/mongodb-redux-common/app-registry';
 import reducer from '../modules';
 
+// TODO: replace this with state coming from the right layer.
```

Comment:
Ticket?

Reply:
I just removed this TODO, I don't think we will ever change dataService being passed to plugins in `configureStore` method one way or another, so this will probably stay here forever anyway